mkdocs.yml

```diff
@@ -36,7 +36,7 @@ nav:
     - Publish and fetch packages: 'guides/publish_fetch_packages.md'
     - Register packages on-chain: 'guides/register_packages_on_chain.md'
     - Deploy a service: 'guides/deploy_service.md'
-    - Running a service in different networks: 'guides/running_service_different_networks.md'
+    - Configure the service blockchain: 'guides/configure_service_blockchain.md'
```

Comment:
Can `blockchain` be confused with the tm blockchain here?

Reply:
Changed section title.